tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "allowSyntheticDefaultImports": true,
     "noImplicitAny": true,
     "experimentalDecorators": true,
-    "typeRoots": ["./server/@types", "./node_modules/@types"]
+    "typeRoots": ["./server/@types", "./node_modules/@types"],
+    "lib": ["ES2020.Promise"]
```

Comment:
I added this in as it allows use of the `Promise.allSettled()` - which is useful when stacking api calls and handling individual failures. Didnt use it in the end as I went down the synchronous route